content/js/record.md

```diff
@@ -1,307 +1,331 @@
 <a name="overview"></a>
 ## Overview
 
-Please make sure you know about and have already configured your skygear
-[container](/js/guide#set-up-app) before you proceed.
+Before proceeding, make you have already configured your
+[skygear container](/js/guide#set-up-app).
 
-<a name="record"></a>
 ### The Record Class
 
-- `Record` must have a type.
-- Each `Record` object is like a dictionary with keys and values; keys will be
-mapped to database column names, and values will be stored appropriately
-based on the data type. Please refer to [Data Type](#data-type)
-section within this guide for more information.
-- `Record` will be owned by the currently logged in user.
-- Each `Record` object has its unique `id` (combination of record type
-  and uuid used in the database as `_id`).
-- `Record` has reserved keys that cannot be used, such as `id` and `_id`.
-Please refer to [Reserved Columns](#reserved) section for more.
+- Record objects are like dictionaries with keys and values; keys will be
```

Comment:
I think It can be clearer if we give an example object, along with a sample database table